WalletWasabi/WabiSabi/Client/CoinJoinCoinSelector.cs

```diff
@@ -49,6 +51,11 @@ public static class CoinJoinCoinSelector
 			.Where(x => x.EffectiveValue(parameters.MiningFeeRate) > Money.Zero)
 			.ToArray();
 
+		if (!filteredCoins.Any())
+		{
+			return ImmutableList<TCoin>.Empty;
```

Comment:
Add LogInfo => No coins suitable for mixing for this round.

Add comments as well.